src/services/recipient.js

```diff
@@ -410,6 +412,29 @@ export async function getGoalsByActivityRecipient(
     ],
     where: goalWhere,
     include: [
+      {
+        model: GoalTemplateFieldPrompt,
+        as: 'prompts',
+        attributes: [
+          ['id', 'promptId'],
+          'ordinal',
+          'title',
+          'prompt',
+          'hint',
+          'fieldType',
+          'options',
+          'validations',
+        ],
+        required: false,
+        include: [
```

Comment:
Instead of including the GoalFieldResponce use the through attribute on the GoalTemplateFieldPrompt. This way it will only match a single value instead of the response for every goal.

Reply:
In this method the whole included would be replaced with the following
through: { attributes:['response'] ],

The required is implied as to get to the prompt it already have to link through the response

Reply:
thank you